packages/nimble-components/src/button/tests/button.spec.ts

```diff
@@ -1,11 +1,56 @@
+import { html } from '@microsoft/fast-element';
 import { Button, buttonTag } from '..';
+import { fixture, type Fixture } from '../../utilities/tests/fixture';
+import { waitForUpdatesAsync } from '../../testing/async-helpers';
 
 describe('Button', () => {
+    async function setup(tabIndex?: string): Promise<Fixture<Button>> {
+        return fixture<Button>(
+            html`<${buttonTag} ${tabIndex ? `tabindex=${tabIndex}` : ''}></${buttonTag}>`
```

Comment:
returning undefined / null to a fast attribute binding removes it which is my guess as to the funky syntax here. can you use a normal fast template?


```suggestion
            html`<${buttonTag} tabindex="${tabindexAttribute}"></${buttonTag}>`
```

See: https://www.fast.design/docs/fast-element/declaring-templates#attributes

> Example: Nullish value

> Some cases may occur where an attribute needs to have a value when used however not present if unused. These are different than boolean attributes by where their presence alone triggers their effect. In this situation, a nullish value (null or undefined) may be provided so the attribute won't exist in that condition.

Reply:
The above comment would have us not parameterizing the fixture for this use-case so may be moot